lib/__init__.py

```diff
@@ -0,0 +1,19 @@
+__version__ = '1.1.0'
```

Comment:
Maybe I can learn something here. If we just remove __init__ does this not just include everything in lib? This was changed by @finfinack in a previous ef1a05917f8445d82c77002cc416227e4d992d84 and things seem to work well enough without it.